applications/nrf_desktop/README.rst

```diff
@@ -952,7 +952,7 @@ nRF52840 Gaming Mouse (nrf52840gmouse_nrf52840)
         * The application is configured to act as a gaming mouse, with both Bluetooth LE and USB transports enabled.
         * Bluetooth is configured to use Nordic's SoftDevice link layer.
 
-      * |preconfigured_build_types|
+      * The configuration with bootloader is set as default.
```

Comment:
```suggestion
      * The configuration with the B0 bootloader is set as default if it fits in the memory.
```
Shouldn`t this be mentioned in "Board configuration" section for nrf5340dk_nrf5340_cpuapp as a note?

Reply:
Good catch, Divya! We need an entry about nRF5340 in both the Requirements table and under nRF Desktop board configuration files. @MarekPieta is telling me nRF5340 is used as nRF Desktop central.